zipkin/src/test/java/zipkin/internal/DependencyLinkerTest.java

```diff
@@ -182,6 +182,18 @@ public void cannotLinkSingleSpanWithoutBothServiceNames() {
   }
 
   @Test
+  public void doesntLinkUnrelatedSpansWhenMissingRootSpan() {
+    List<DependencyLinkSpan> trace = asList(
```

Comment:
in tests, most of the time, we use incrementing numbers to make obvious the order in the tree. This confused me a little as 3 is the parent of 1 and 2.

Maybe rework a little?

long parentId = 1L; // by declaring a local, it is obvious that the parent is missing. It is more obvious because the traceId low is the same as the parent id.
// then use (parentId, 2L) and (parentId, 3L)